### DISCUSSION TITLE
RainbowKit is expanding access to the  @Safe  app ecosystem

### DISCUSSION THREAD

**Discussion by tiwarishankar** [rainbow-me/rainbowkit#1116] [Q&A] [answered]
How we can intergrate safe with Rainbowkit ??? any documentation

**Comment by DanielSinclair** [comment, marked as answer]
RainbowKit dApps that use `getDefaultWallets` will work by default within the Safe browser as long as those apps include a `manifest.json` at the root according to the [Safe Apps SDK](https://docs.safe.global/safe-core-aa-sdk/safe-apps/release) guide. You'll also want to submit your dApp to the Safe App registry for easy access for end-users.
